src/components/GlobalPreferences/GlobalPreferences.js

```diff
@@ -65,52 +66,57 @@ function GlobalPreferences({
   useEsc(onClose)
 
   return (
-    <div>
-      <Close onClick={isSharedLink ? handleSharedIdentitiesClose : onClose} />
-      <Layout>
-        <Header
-          primary={isSharedLink ? 'Save Custom Labels' : 'Global preferences'}
+    <Layout>
+      <Close
+        compact={compact}
+        onClick={isSharedLink ? handleSharedIdentitiesClose : onClose}
+      />
+      <Header
+        primary={isSharedLink ? 'Save Custom Labels' : 'Global preferences'}
+        css={`
+          padding-top: ${!compact ? 10 * GU : 0}px;
+        `}
+      />
+      {isSharedLink ? (
+        <SharedIdentities
+          isSaving={isSavingSharedLink}
+          onSave={handleSharedIdentitiesSave}
+          onCancel={handleSharedIdentitiesCancel}
+          identities={sharedIdentities}
+          onToggleAll={handleSharedIdentitiesToggleAll}
+          onToggleIdentity={handleSharedIdentitiesToggleIdentity}
+          selected={sharedIdentitiesSelected}
+          allSelected={sharedIdentitiesAllSelected}
+          someSelected={sharedIdentitiesSomeSelected}
         />
-        {isSharedLink ? (
-          <SharedIdentities
-            isSaving={isSavingSharedLink}
-            onSave={handleSharedIdentitiesSave}
-            onCancel={handleSharedIdentitiesCancel}
-            identities={sharedIdentities}
-            onToggleAll={handleSharedIdentitiesToggleAll}
-            onToggleIdentity={handleSharedIdentitiesToggleIdentity}
-            selected={sharedIdentitiesSelected}
-            allSelected={sharedIdentitiesAllSelected}
-            someSelected={sharedIdentitiesSomeSelected}
+      ) : (
+        <React.Fragment>
+          <TabBar
```

Comment:
```suggestion
          <Tabs
```

Reply:
Done in 3446afb since it required renaming the import too.